fhirResourcesToLoad/0. pharmacy.json

```diff
@@ -0,0 +1,28 @@
+{
+  "resourceType": "HealthcareService",
+  "id": "pharm0111",
+  "active": true,
+  "name": "Test Pharmacy",
+  "category": [
+    {
+      "coding": [
+        {
+          "system": "http://terminology.hl7.org/CodeSystem/service-category",
+          "code": "17",
+          "display": "Pharmacy"
+        }
+      ]
+    }
+  ],
+  "type": [
```

Comment:
Service Type pharmacy from the service-type ValueSet is 64 from https://hl7.org/fhir/R4/valueset-service-type.html
[http://terminology.hl7.org/CodeSystem/service-type](https://hl7.org/fhir/R4/codesystem-service-type.html)